tests/constants/admincli/cmd.go

```diff
@@ -79,4 +79,15 @@ const (
 
 	//RemoveVolumes option refers to removing all volumes from a vmgroup
 	RemoveVolumes = " --remove-volumes"
+
+	// CreateVsanPolicy referring to create vsanPolicy
+	// where --name will be name of the vsanPolicy
+	CreateVsanPolicy = vmdkopsAdmin + "policy create --name="
```

Comment:
It is being used. we had a separate PR to introduce a common utils. The change that I merged yesterday had a test to create volume with vsan policy. That uses CreatePolicy